app/javascript/angular/code/services/_draw_session.js.erb

```diff
@@ -101,7 +101,11 @@ const createMeasurementMarker = (measurement, idx, heat, map, suffix) => {
     position: { lat: measurement.latitude, lng: measurement.longitude },
     title: roundedValue.toString() + suffix,
     zIndex: idx,
-    icon: eval(`locationMarker${level}Path`)
+    icon: {
+      anchor: new google.maps.Point(6, 6),
+      size: new google.maps.Size(12, 12),
+      url: eval(`locationMarker${level}Path`)
```

Comment:
Can we avoid using eval? Wouldn't the internal string be enough?

Reply:
the internal string is the variable name we're generating :) so we have to eval it to get the string it keeps

Reply:
cos it's assigned to `"<%= helpers.image_path('location_marker4.svg') %>"` etc., I think I'd still need eval to use it

Reply:
Not sure about that. But it's a minor thing (and it was already there) ;)